packages/bot-web-ui/src/components/flyout.jsx

```diff
@@ -107,11 +108,12 @@ const FlyoutContent = props => {
                             }
                             case Blockly.Xml.NODE_BUTTON: {
                                 const cb_key = node.getAttribute('callbackKey');
-                                const button_cb = Blockly.derivWorkspace.getButtonCallback(cb_key);
-                                const callback = button_cb || (() => {});
+                                const cb_id = node.getAttribute('id');
```

Comment:
Is `cb` callback? If so, can we rename `cb_id` to `callback_id` for clarity? As well as `cb_key`.

Reply:
Done 👍